src/svg2pdf.js

```diff
@@ -2190,6 +2190,14 @@ SOFTWARE.
         _pdf.setGState(new _pdf.GState(gState));
       }
 
+      var isNodeFillRuleEvenOdd = false
```

Comment:
How about just
````
var isNodeFillRuleEvenOdd = getAttribute(node, "fill-rule") === "evenodd";
````
And is there a reason we don't do this in the nodeIs() branch, only? "getAttribute" is relatively expensive because of the inheritance CSS stuff and we might only want to evaluate it, if really necessary.

Reply:
That works! I was just following the pattern of some of the other getAttribute checks - but moving it into the nodeIs() branch makes sense.

Reply:
@yGuy updated

Reply:
Looks good to me!